runtime/commonMain/src/kotlinx/serialization/Annotations.kt

```diff
@@ -99,3 +99,15 @@ public annotation class UseSerializers(vararg val serializerClasses: KClass<out
  */
 @Target(AnnotationTarget.PROPERTY, AnnotationTarget.TYPE, AnnotationTarget.CLASS)
 public annotation class Polymorphic
+
+
+/**
+ * Public API marked with this annotation is effectively **internal**, which means
```

Comment:
Should we mention the reason it still has public modifier? Docs without it may leave a user with a question 'why they just didn't use `internal` modifier'

Reply:
Ktor and coroutines experience shows that it's not really necessary.